e2e/cypress/integration/system_console/environment_spec.js

```diff
@@ -284,6 +284,20 @@ describe('Environment', () => {
         });
     });
 
+    it('MM-T963 Configuration - Purge caches', () => {
+        cy.visit('/admin_console/environment/web_server');
+
```

Comment:
Thanks @alejandrovelez7  all look good, test passed for me.
Just few small changes to follow standard. 
As mentioned at the top of spec file, can you please add comments using following style,
`#` indicates a test step (e.g. # Go to a page)
`*` indicates an assertion (e.g. * Check the title)

For reference, see MM-T959 in the same spec file.

Reply:
Updated in https://github.com/mattermost/mattermost-webapp/pull/9023/commits/a7ed7af529f8ff553162d41776b588051ad3fd17 let me know what you think!